shell/i18n/en.i18n.json

```diff
@@ -344,7 +345,42 @@
           "hasApps": "Select an app and create your first grain.",
           "hasNoApp": "Install an app to create your first grain."
         },
-        "moveToTrash": "Move to trash"
+        "transfers": {
+          "massTransfer": "Mass transfer...",
+          "transfers": "Transfers",
+          "transfer": "Transfer",
+          "blockedInitiateBeforeSource": "It looks like you're trying to initiate a new transfer from",
+          "blockedInitiateAfterSource": ", but you already have a transfer in progress. Please complete or cancel the existing transfer to continue.",
+          "incomingTransferActive": "Incoming transfer active",
+          "cancel": "Cancel",
+          "remaining": "Remaining",
+          "retryErrors": "Retry Errors",
+          "pauseTransfer": "Pause Transfer",
+          "continueTransfer": "Continue Transfer",
+          "startTransfer": "Start Transfer",
+          "closeTransfer": "Close Transfer",
+          "outgoingTransferActive": "Outgoing transfer active",
+          "currentlyTransferring": "You are currently transferring grains to:",
+          "manageAt": "Manage your transfer at:",
+          "inOrder": "In order to start a new transfer (incoming or outgoing), you must cancel the existing one.",
+          "cancelTransfer": "Cancel Transfer",
+          "initiateInboundTransfer": "Initiate Inbound Tranfer",
+          "initiateOutboundTransfer": "Initiate Outbound Tranfer",
```

Comment:
Here on both `initiateInboundTransfer` and `initiateOutboundTransfer`, you introduced a new misspelling of transfer, Tranfer.